packages/animation/src/components/provider.js

```diff
@@ -37,9 +37,9 @@ if (!('KeyframeEffect' in window)) {
 /**
  * Internal dependencies
  */
-import StoryPropTypes from '../../edit-story/types';
+import StoryPropTypes from '../../../../assets/src/edit-story/types';
```

Comment:
Not sure what to do about this one.

Reply:
As mentioned on the ticket, ideally we'd import these types from a dedicated package.

For the time being, we could just _copy_ the whole proptype from `edit/story/types.js` to this file. Adding a `TODO` comment indicating that this should be improved in the future.

e.g.

```js
const StoryElementPropType = PropTypes.shape({
  id: PropTypes.string.isRequired,
  type: PropTypes.string.isRequired,
  x: PropTypes.number.isRequired,
  y: PropTypes.number.isRequired,
  width: PropTypes.number.isRequired,
  height: PropTypes.number.isRequired,
  flip: StoryPropTypes.flip,
  rotationAngle: PropTypes.number.isRequired,
  mask: StoryPropTypes.mask,
  link: StoryPropTypes.link,
  opacity: PropTypes.number,
  lockAspectRatio: PropTypes.bool,
});
```

Reply:
Fixed